_episodes/documentation.md

```diff
@@ -1,33 +1,92 @@
+- "Evaluate existing data documentation with a critical view towards reusability"
+- "Ensure that others can re-use your data by writing clear documentation."
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+- "Documentation provides rich contextual information mostly intended for human readers."
+- "Documentation is essential for data reuse and reproducibility."
 ---
 
+## What is documentation?
 
-Documentation is essential for understanding and (re) using data:
+![https://xkcd.com/833/](https://imgs.xkcd.com/comics/convincing.png)
+*Source: <https://xkcd.com/833/>*
 
-Data specific information: what does this number represent, what units is it in, is it a time average, what is the relation between 2 datapoints (subsequent in time, distinct in space, same or different variable), etc.
+Data documentation is all relevant information needed to properly interpret a
+dataset. Documentation can be very specific about the data. For example, what
+does this dataset represent, what units is it in, is it a time average, what is
+the relation between 2 datapoints (subsequent in time, distinct in space, same
+or different variable), etc. But, the documentation can also answer some
+questions about the project: why were these data collected, by whom, and what
+questions do they address? Was there any kind of quality assurrence?
 
-Project information: how was the data obtained (where there steps)? By whom? Are there different versions of the data? Quality assurance?
+## Where to document my data?
 
-On data specific info you often find difficult terms like ‘semantics’, ‘vocabularies’, ‘ontologies’, ‘linked data’. In climate (related) science, these are often so ‘logical’ that you won’t even notice they’re there. However, look at this example of a patient’s data model, and the netcdf model. Do you see it now? More info on this in sections file format and metadata.
+Documentation can take many forms: a data management plan, papers published,
+(jupyter) notebooks, lab journals, provenance information, etc. It is a good
+idea to bundle these with the data when you publish the data, so that people
+that find your data will not have to search for it elsewhere. Most data
+repositories have a 'description' field, which is a suitable place to provide
+any information that is not covered by the other metadata fields.
 
-Project info is typically written in the data description fields, but may also include pointers to external documents, such as the data management plans.
+> ## Grouping related files with Zenodo communities
+> Data documentation in your project may be scattered over many files of
+> different types. To bundle this information, Zenodo provides 'communities'.
+> You can upload all your files to Zenodo and add them to a community. You can
+> also track versions. Each file gets its own DOI. So you could e.g. cite your
+> data management plan in the description field of your dataset.
+{: .callout}
```

Comment:
```suggestion
> Data documentation in your project may be scattered over many files of
> different types. To bundle this information, Zenodo allows you to upload
> a set of related files (hundreds if needed) as a single dataset.
> 
> In addition, Zenodo provides 'communities'. You can upload all your datasets
> to Zenodo and add them to a community, thus keeping track of all relevant
> data for an entire project. You can also track versions. Each dataset
> gets its own DOI. So you could e.g. cite your data management plan
> in the description field of your dataset.
{: .callout}
```

Reply:
Zenodo is not introduced at all. Is this already discussed in a previous lesson?

Zenodo also allows multiple files for a single dataset. Changed the text, but perhaps it is too complex now.

Reply:
I like it, thanks!